Dockerfile

```diff
@@ -1 +1,32 @@
-FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/base_centos:7-20210630
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/centos/base-image:java8
+
+ENV SOLR_VERSION 7.7.2
+
+ENV SOLR_URL https://archive.apache.org/dist/lucene/solr/$SOLR_VERSION/solr-$SOLR_VERSION.tgz
+
+ENV SOLR_PORT 8983
+
+ADD $SOLR_URL /tmp
+
+RUN useradd -M solr && \
+    cd /tmp && \
+    yum install lsof -y && \
+    tar -xzf solr-$SOLR_VERSION.tgz -C /opt && \
+    rm solr-$SOLR_VERSION.tgz && \
+    cd /opt && \
+    mv solr-$SOLR_VERSION solr && \
+    chown -R solr:solr /opt/solr && \
```

Comment:
Please `chown -R root:root /opt/solr` and give Solr write access only where necessary. I believe that will only be for the index, which will be externally mounted anyway...

Also, why aren't we using the `/app` working directory in order to be consistent with other services?